src/service/RewardProgramService.ts

```diff
@@ -119,7 +119,7 @@ export class RewardProgramService implements TransactionFactory {
             return transactions;
         }
         const agentUrl = nodePreset.agentUrl || `https://${nodePreset.host}:7880`;
```

Comment:
The default port will be changed to `7881` instead (with this PR => https://github.com/nemgrouplimited/symbol-node-rewards/pull/234)